export/csv/wormhole.csv

```diff
@@ -1,81 +1,81 @@
+"2";"A239";"L";"2000000000";"300000000";;"24";"5";
+"3";"A641";"H";"2000000000";"1000000000";;"16";"10";
+"4";"A982";"C6";"3000000000";"300000000";;"24";"2.22";
+"5";"B041";"C6";"3000000000";"300000000";"500000000";"48";;
+"6";"B274";"H";"2000000000";"300000000";;"24";"10";
+"7";"B449";"H";"2000000000";"1000000000";;"16";;
+"8";"B520";"H";"3000000000";"300000000";"500000000";"24";;
+"9";"C008";"C5";;"5000000";"1";"16";;
+"10";"C125";"C2";"1000000000";"20000000";;"16";"6.67";
+"11";"C140";"L";"3000000000";"1350000000";;"24";"5";
+"12";"C247";"C3";"2000000000";"300000000";;"16";"10";
+"13";"C248";"0.0";"3000000000";"1800000000";"500000000";"24";;
+"14";"C391";"L";"3000000000";"1800000000";"500000000";"24";;
+"15";"D364";"C2";"1000000000";"300000000";;"16";"1.25";
+"16";"D382";"C2";"2000000000";"300000000";;"16";"6.67";
+"17";"D792";"H";"3000000000";"1000000000";;"24";"2.5";
+"18";"D845";"H";"3000000000";"300000000";"500000000";"24";"5";
+"19";"E004";"C1";;"5000000";"1";"16";;
+"20";"E175";"C4";"2000000000";"300000000";;"16";"5";
+"21";"E545";"0.0";"2000000000";"300000000";;"24";"2.5";
+"22";"G009";"C6";;"5000000";;"16";;
+"23";"G024";"C2";"2000000000";"300000000";;"16";"1.25";
+"24";"H121";"C1";"500000000";"20000000";;"16";"10";
+"25";"H296";"C5";"3000000000";"1350000000";;"24";"10";
+"26";"H900";"C5";"3000000000";"300000000";;"24";"2.5";
+"27";"I182";"C2";"2000000000";"300000000";;"16";"4";
+"28";"J244";"L";"1000000000";"20000000";;"24";"5";
+"29";"K329";"0.0";"3000000000";"1800000000";"500000000";"24";;
+"30";"K346";"0.0";"3000000000";"300000000";;"24";"2.5";
+"31";"L005";"C2";;"5000000";"1";"16";;
+"32";"L477";"C3";"2000000000";"300000000";;"16";"5";
+"33";"L614";"C5";"1000000000";"20000000";;"24";"2.5";
+"34";"M006";"C4";;"5000000";"1";"16";;
+"35";"M267";"C3";"1000000000";"300000000";;"16";"1.25";
+"36";"M555";"C5";"3000000000";"1000000000";;"24";"2.5";
+"37";"M609";"C4";"1000000000";"20000000";;"16";"4";
+"38";"N062";"C5";"3000000000";"300000000";;"24";"2.5";
+"39";"N110";"H";"1000000000";"20000000";;"24";"10";
+"40";"N290";"L";"3000000000";"1800000000";"500000000";"24";;
+"41";"N432";"C5";"3000000000";"1350000000";;"24";"10";
+"42";"N766";"C2";"2000000000";"300000000";;"16";"4";
+"43";"N770";"C5";"3000000000";"300000000";;"24";"2.5";
+"44";"N944";"L";"3000000000";"1350000000";;"24";"10";
+"45";"N968";"C3";"2000000000";"300000000";;"16";"10";
+"46";"O128";"C4";"1000000000";"300000000";"100000000";"24";;
+"47";"O477";"C3";"2000000000";"300000000";;"16";"5";
+"48";"O883";"C3";"1000000000";"20000000";;"16";"5";
+"49";"P060";"C1";"500000000";"20000000";;"16";"5";
+"50";"Q003";"0.0";;"5000000";"1";"16";;
+"51";"Q317";"C1";"500000000";"20000000";;"16";"2.5";
+"52";"R051";"L";"3000000000";"1000000000";;"16";"5";
+"53";"R474";"C6";"3000000000";"300000000";;"24";"2.22";
+"54";"R943";"C2";"750000000";"300000000";;"16";"6.67";
+"55";"S047";"H";"3000000000";"300000000";;"24";;
+"56";"S199";"0.0";"3000000000";"1350000000";;"24";;
```

Comment:
According to http://wiki.eve-inspiracy.com/index.php?title=Wormhole_Signature_Strength_List **S199** has a strength of 10%